common/Util.cpp

```diff
@@ -185,15 +185,15 @@ namespace Util
 
     std::string replace(std::string result, const std::string& a, const std::string& b)
     {
-        const std::size_t aSize = a.size();
-        if (aSize > 0)
+        const std::size_t sizeA = a.size();
```

Comment:
Nit: I'm not sure this is an improvement here, "a" and "b" are not Hungarian notation for "anything" and "boolean", but refer to the size of "a" and "b". What about a follow-up PR to rename these further to something like: "a" -> "from", "b" -> "to", and then you can go back to fromSize and toSize?